assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -230,7 +243,7 @@ function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
       <SeparateBorder />
       <WithTooltip title={__('Align left', 'web-stories')} shortcut="mod+{">
         <IconButton
-          disabled={!isAlignEnabled}
+          disabled={isFill}
```

Comment:
I still think this should be moved to a variable above - it might read very weird to a future developer, that this button is disabled because of a completely unrelated feature here.

Reply:
Fixed!